src/main/scala/com/typesafe/sbt/license/TargetLanguage.scala

```diff
@@ -77,3 +77,19 @@ case object Html extends TargetLanguage {
 
   def htmlEncode(s: String) = org.apache.commons.lang3.StringEscapeUtils.escapeHtml4(s)
 }
+
+case object Csv extends TargetLanguage {
+  val ext = "csv"
+  def documentStart(title: String, reportStyleRules: Option[String]): String = ""
+  def documentEnd(): String = ""
+  def createHyperLink(link: String, content: String): String = s"$content"
```

Comment:
I'd like to persuage you to include the hyperlink somehow... Perhaps as `s"$content ($link)"`.